### ISSUE TITLE
dom-repeat inside a dom-bind not working

### ISSUE THREAD

**Issue by kevinpschaaf** [Polymer/polymer#3911]
_From @valdrinkoshi on July 29, 2016 20:21_

Binding is broken between the parent `dom-bind` and the child `dom-repeat` 
In alacarte http://jsbin.com/gapowo/2/edit?html,output:

``` html
  <dom-bind id="domBind">
    <template>
      <dom-repeat id="domRepeat" items="{{items}}">
        <template>
          <div id$="[[item.name]]">{{item.name}}</div>
        </template>
      </dom-repeat>
    </template>
  </dom-bind>
  <script>
    domBind.items = [{name:'item0'}, {name: 'item1'}, {name: 'item2'}, {name: 'item3'}];
  </script>
```

In 1.0 http://jsbin.com/decate/3/edit?html,output:

``` html
  <template is="dom-bind" id="domBind">
    <template is="dom-repeat" id="domRepeat" items="{{items}}">
      <div id$="[[item.name]]">{{item.name}}</div>
    </template>
  </template>
  <script>
    domBind.items = [{name:'item0'}, {name: 'item1'}, {name: 'item2'}, {name: 'item3'}];
  </script>


_Copied from original issue: PolymerLabs/alacarte#15_
```

**Comment by kevinpschaaf** [comment]
_From @blasten on August 9, 2016 18:48_

I'm having the same issue. It seems like a timing issue, maybe time to upgrade? If set the value in the next tick then it works. e.g.
```js
setTimeout(_ => {
  domBind.items = ...;
}, 0);
```

**Comment by kevinpschaaf** [comment]
_From @blasten on August 9, 2016 19:24_

As a side effect, l[ookups for listener methods fail the first time](https://github.com/PolymerLabs/alacarte/blob/a8fd573a211836d6fc38020b9138a8d1bd2b2a0b/src/events/event-listeners.html#L28) if they are used within a dom-bind.

**Comment by kevinpschaaf** [comment]
Resolved on 2.0-preview. Working jsbin: http://jsbin.com/nekebinolu/1/edit
